KeyboardKit/KeyboardShortcuts.swift

```diff
@@ -0,0 +1,52 @@
+#if canImport(SwiftUI)
+import SwiftUI
+
+@available(iOS 14.0, *)
+public extension KeyboardShortcut {
```

Comment:
I’m a bit concerned about publicly extending a UIKit class and adding fairly generically named properties to it. Need to check what actually happens if an app was to implement a similar extension with properties with the same name. But maybe it’s fine.

Reply:
Fair point – testing this out in a simple project, the compiler (Swift 5.7 / Xcode 14) doesn't give any warning or error. My locally defined extension was used in place of KeyboardKit's of the same name